awsagentprovider/src/main/java/software/amazon/opentelemetry/javaagent/providers/AwsApplicationSignalsCustomizerProvider.java

```diff
@@ -70,6 +71,8 @@
 public class AwsApplicationSignalsCustomizerProvider
     implements AutoConfigurationCustomizerProvider {
   static final String AWS_LAMBDA_FUNCTION_NAME_CONFIG = "AWS_LAMBDA_FUNCTION_NAME";
+  private static final String XRAY_OTLP_ENDPOINT_PATTERN =
+      "https://xray\\.([a-z0-9-]+)\\.amazonaws\\.com/v1/traces$";
```

Comment:
Minor improvement: Can include `^` to enforce "http" is at the start of the endpoint string
```suggestion
      "^https://xray\\.([a-z0-9-]+)\\.amazonaws\\.com/v1/traces$";
```